common-protobuf/src/test/java/io/protostuff/runtime/TestProtobufCompatibleUtils.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.protostuff.runtime;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import io.protostuff.ByteArrayInput;
+import io.protostuff.LinkedBuffer;
+import io.protostuff.ProtobufOutput;
+import io.protostuff.runtime.model.ModelProtobuf;
+import io.protostuff.runtime.model.ModelProtostuff;
+import io.protostuff.runtime.model.User;
+
+public class TestProtobufCompatibleUtils {
+    @Test
+    public void test() throws IOException {
```

Comment:
meaningful test method names really help readers understand what the test is about